lib/perl/Genome/Model/Tools/Transcriptome/ErccMapUnaligned.t

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env genome-perl
+
+# P R A G M A S ###############################################################
+use strict;
+use warnings;
+
+# M O D U L E S ###############################################################
+use above 'Genome';
+use Path::Class;
+use Test::More;
+
+# M A I N #####################################################################
+
+#This is a bare minimum test that just compiles Perl and the UR class.
+use_ok('Genome::Model::Tools::Transcriptome::ErccMapUnaligned');
+done_testing();
+
+
+# This is an integration test that relies on data in production
```

Comment:
I thought it would be good to check that the module at least compiles, but I can go ahead and make that change.